tests/test_acousticbrainz_genre.py

```diff
@@ -55,3937 +35,29 @@ def test_track(httpserver):
-                    23.580953598,
-                    11.0932693481,
-                    3.13923668861,
-                ],
-                [
-                    -16.950138092,
-                    37.7287139893,
-                    -17.4020252228,
-                    2.53039526939,
-                    -42.4019470215,
-                    -29.1598892212,
-                    -6.63176584244,
-                    5.81215953827,
-                    51.1233444214,
-                    106.574928284,
-                    59.1461105347,
-                    23.1877384186,
-                    5.50347471237,
-                ],
-                [
-                    -48.6743812561,
-                    31.5271949768,
-                    -33.5541763306,
-                    9.81129646301,
-                    -8.1498708725,
-                    -30.9757270813,
-                    -2.64764499664,
-                    6.34349679947,
-                    23.580953598,
-                    59.1461105347,
-                    103.604682922,
-                    46.4150886536,
-                    9.88276195526,
-                ],
-                [
-                    47.0859146118,
-                    -30.5362644196,
-                    -60.2577857971,
-                    8.50957870483,
-                    -10.4739961624,
-                    13.9199514389,
-                    2.61026930809,
-                    7.00799417496,
-                    11.0932693481,
-                    23.1877384186,
-                    46.4150886536,
-                    85.4878540039,
-                    39.7927474976,
-                ],
-                [
-                    -9.6302986145,
-                    -58.9328155518,
-                    -1.63439774513,
-                    -11.8001689911,
-                    18.9017391205,
-                    31.1762313843,
-                    23.4787330627,
-                    8.87113571167,
-                    3.13923668861,
-                    5.50347471237,
-                    9.88276195526,
-                    39.7927474976,
-                    76.2382049561,
-                ],
-            ],
-        },
-        "spectral_decrease": {
-            "dmean2": 5.08372233199e-09,
-            "median": -5.58059509714e-09,
-            "min": -5.60004380645e-08,
-            "dvar2": 4.32927042758e-17,
-            "dvar": 1.93053019642e-17,
-            "dmean": 3.32266036907e-09,
-            "max": 2.9793302381e-18,
-            "var": 7.8599522368e-17,
-            "mean": -9.43999456382e-09,
-        },
-        "barkbands_crest": {
-            "dmean2": 5.16687965393,
-            "median": 12.3140182495,
-            "min": 2.29649472237,
-            "dvar2": 21.2763710022,
-            "dvar": 8.37867069244,
-            "dmean": 3.1435611248,
-            "max": 25.9979152679,
-            "var": 29.487985611,
-            "mean": 13.2201347351,
-        },
-    },
-}
```

Comment:
it was being used - it was doing a very strict test(making sure the full loaded json matched) but in the end, it was effectively testing if json.load is working properly. Since I really hope it is 😁  I took this test out and instead just had it check that it loads a dictionary.